src/vue/backend/src/views/settings/General.vue

```diff
@@ -2,12 +2,13 @@
     <TwoColumn title="General Settings">
         <APIKey
             :apiKey="localStateValues.apiKey"
+            :disabled="formDisabled"
```

Comment:
Same as comment above about using the singular variable in both places so that you have the loading animation and the disabled form